modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/ZoneResourcesManager.java

```diff
@@ -98,13 +109,14 @@ private ZoneResources(TxStateStorage txStateStorage) {
     }
 
     @TestOnly
+    @Nullable
     public TxStatePartitionStorage txStatePartitionStorage(int zoneId, int partitionId) {
         ZoneResources resources = resourcesByZoneId.get(zoneId);
-        assert resources != null : "No resources yet for zone " + zoneId;
 
-        TxStatePartitionStorage partitionStorage = resources.txStateStorage.getPartitionStorage(partitionId);
-        assert partitionStorage != null : "No partition storage for zone " + zoneId + " and partition " + partitionId;
+        if (resources == null) {
```

Comment:
I couldn't understand why did you change the method in this way?

Reply:
This is test-only method. Previously it threw an assertion if resources didn't exist. In order to write a unit test, I need it to return `null` instead.